src/Provider/SessionServiceProvider.php

```diff
@@ -10,366 +10,253 @@
     {
-        if (isset($options['host']) || isset($options['port'])) {
-            $options['connections'][] = $options;
-        } elseif ($options['connection']) {
-            $options['connections'][] = $options['connection'];
-        }
-
-        /** @var ParameterBag[] $toParse */
-        $toParse = [];
-        if (isset($options['connections'])) {
-            foreach ((array) $options['connections'] as $alias => $conn) {
-                if (is_string($conn)) {
-                    $conn = ['host' => $conn];
-                }
-                $conn += [
-                    'scheme' => 'tcp',
-                    'host'   => $defaultHost,
-                    'port'   => $defaultPort,
-                ];
-                $conn = new ParameterBag($conn);
-
-                if ($conn->has('password')) {
-                    $conn->set('pass', $conn->get('password'));
-                    $conn->remove('password');
-                }
-                $conn->set('uri', Uri::fromParts($conn->all()));
-
-                $toParse[] = $conn;
-            }
-        } elseif (isset($options['save_path'])) {
-            foreach (explode(',', $options['save_path']) as $conn) {
-                $conn = new ParameterBag($conn);
-                $conn->set('uri', new Uri($conn));
-                $toParse[] = $conn;
-            }
-        }
-
-        $connections = [];
-        foreach ($toParse as $conn) {
-            /** @var Uri $uri */
-            $uri = $conn->get('uri');
-
-            $parts = explode(':', $uri->getUserInfo(), 2);
-            $password = isset($parts[1]) ? $parts[1] : null;
-
-            $connections[] = [
-                'scheme'     => $uri->getScheme(),
-                'host'       => $uri->getHost(),
-                'port'       => $uri->getPort(),
-                'path'       => $uri->getPath(),
-                'alias'      => $conn->get('alias'),
-                'prefix'     => $conn->get('prefix'),
-                'password'   => $password,
-                'database'   => $conn->get('database'),
-                'persistent' => $conn->get('persistent'),
-                'weight'     => $conn->get('weight'),
-                'timeout'    => $conn->get('timeout'),
-            ];
-        }
-
-        return $connections;
+        $this->app['session.bag.options'] = $this->app->share(function () {
+            return new OptionsBag([
+                'cluster' => 'redis',
+                'cookie_lifetime' => 0,
+                'connections' => [
+                    [
+                        'host' => '127.0.0.1',
+                        'port' => false
+                    ]
+                ],
+                'database' => 0,
+                'dir' => false,
+                'gc_divisor' => 100,
+                'gc_maxlifetime' => 1440,
+                'gc_propability' => 1,
+                'handler' => 'filesystem',
+                'lazy_write' => false,
```

Comment:
this was the default where it's used, in `Bolt\Session\SessionStorage`. @CarsonF may be able to chime in.